src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
Passing expressions directly in the constructor can reduce readability. It's better to create a separate variable with a descriptive name for the radius value and then pass it to the constructor.